spec/example_app/app/controllers/docs_controller.rb

```diff
@@ -87,7 +96,7 @@ def title
     end
 
     def title_suffix
-      metadata["home"] ? "" : " - Administrate"
+      metadata["home"] ? "Administrate" : " - Administrate"
```

Comment:
Am I correct that this `metadata["home"]` doesn't exist any more for any page? If so, this line probably needs removing the reference to it.

Reply:
Oh, I see it's added in the constructor. I feel that's a bit too indirect. Would it make sense to directly the following instead, and do away with the "artificial" metadata?

```suggestion
      front_matter_parsed.front_matter.empty? ? "Administrate" : " - Administrate"
```

Reply:
Yeah, I was thinking something like that. So far I've just rebased it, but I sketched out a potentially different way of doing it the other day, which I should be able to get back to soon.